nipoppy/trackers/run_tracker.py

```diff
@@ -97,29 +133,57 @@ def run(global_configs, dash_schema_file, pipelines, session_id="ALL", run_id=1,
                 
                 if dir_status:                
                     for name, func in status_check_dict.items():
-                        status = func(subject_dir, session, run_id)
+                        status = func(subject_dir, session_id, run_id)
                         logger.info(f"task_name: {name}, status: {status}")
                         _df.loc[bids_id,name] = status
                         _df.loc[bids_id,"pipeline_starttime"] = get_start_time(subject_dir)
-                        _df.loc[bids_id,"pipeline_endtime"] = UNAVAILABLE # TODO
+                        _df.loc[bids_id,"pipeline_endtime"] = get_end_time(subject_dir)
```

Comment:
This could get very slow depending on number of files. Maybe we should either only check the files listed in the tracker config or parse the pipeline log to get completion timestamp.